build.gradle

```diff
@@ -38,6 +39,8 @@ dependencies {
     runtimeOnly('org.springframework.boot:spring-boot-devtools')
     testImplementation('org.springframework.boot:spring-boot-starter-test')
     implementation('me.alidg:errors-spring-boot-starter:1.2.0')
+    testImplementation('junit:junit:4.12')
+    testImplementation('org.mockito:mockito-core:2.1.0')
```

Comment:
Mockito is the mocking framework I decided to use. Very simple and familiar interface.